lib/get-config.js

```diff
@@ -31,7 +32,7 @@ async function requirePlugin(workingDir, pluginName, fromConfigPath) {
   let pluginPath = resolve.sync(pluginName, { basedir, extensions: ['.js', '.mjs', '.cjs'] });
 
   try {
-    const { default: plugin } = await import(pluginPath);
+    const { default: plugin } = await import(pathToFileURL(pluginPath));
```

Comment:
Wouldn't we also need this for the config loading on line 84 too?

Reply:
good catch! pushed